justfile

```diff
@@ -0,0 +1,88 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+_default:
+  @just --list
+
+_env:
+  @if [ ! -f .env ]; \
```

Comment:
I think I would have done four space indents. It's a little easier to read, it's what the just examples use, etc. But this is probably fine.

Also, instead of making this a one-liner, you can make it an inline bash script:

https://just.systems/man/en/shebang-recipes.html

Then we don't have to deal with the `\` stuff.